experiments/ClimaCore/sea_breeze/README.md

```diff
@@ -1,66 +1,16 @@
+#=
```

Comment:
I wonder whether we could delete this file, since the `..._rhs.jl` files include the formulations. We can add the flux calculation to `atmos_rhs.jl`, and clarify the coupling under `Component Models` in `run.jl`